llvm/test/DebugInfo/CAS/AArch64/debug_unopt.ll

```diff
@@ -0,0 +1,77 @@
+; RUN: llc -debug-info-unopt -O0 --filetype=obj --cas-backend --cas=%t/cas --mccas-casid %s -o %t/debug_unopt.id
+; RUN: llvm-cas-dump --cas=%t/cas --casid-file %t/debug_unopt.id | FileCheck %s
+
+; CHECK: mc:debug_string_section llvmcas://
+; CHECK-NEXT: mc:debug_string llvmcas://
+; CHECK-NEXT: mc:section      llvmcas://
+
+; CHECK: mc:debug_line_section llvmcas://54091b35d960f7f4f23aeeee6805bc221a943b1b17f624d29b92d36212b271cf
+; CHECK-NEXT: mc:debug_line_unopt llvmcas://24724ef11cfa48466515553ebdd6e8a2e726922d65177ed8751aed487363d571
+; CHECK-NEXT: mc:padding      llvmcas://1dfed76cbca2e08753543cab63fdd0fa0aaf5af34ac85ac53334d00e2e3dc649
+
```

Comment:
I think this test can be simplified quite a bit:

* All the comment lines can be removed
* Do the functions need any instructions? Can they simply `ret void` or just `ret i32 some_number` if it is important they have different bodies?
* Most metadata can be removed, especially the attribute stuff